content/en/get-started/getting-started-with-corda-4.md

```diff
@@ -14,12 +14,12 @@ The best way to get started with Corda is to:
 1. Decide whether you need to use:
    * Corda Enterprise - the enterprise version of Corda, for which you require a commercial contract with R3.
    * Corda open source - the open source version of Corda, which you can build on now.
-   Jump to: [Corda](../platform/corda/4.8/open-source/_index.md) or [Corda Enterprise](../platform/corda/4.8/enterprise/_index.md).
+   Jump to: [Corda](../../en/platform/corda/4.8/open-source.html) or [Corda Enterprise](../../en/platform/corda/4.8/enterprise.html).
 2. Familiarize yourself with the [Corda key concepts](./corda-key-concepts.md).
-3. Run a [sample CorDapp](../tutorials/corda/4.8/os/template-tutorial/writing-a-cordapp-using-a-template.md) to see Corda in action.
-4. [Build your own CorDapp](../tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-intro.md) from scratch.
+3. Run a [sample CorDapp](../../en/tutorials/corda/4.8/os/template-tutorial/writing-a-cordapp-using-a-template.md) to see Corda in action.
```

Comment:
I think this should actually link to this page: https://docs.r3.com/en/platform/corda/4.8/open-source/tutorial-cordapp.html

Reply:
```suggestion
3. Run a [sample CorDapp](../../en/platform/corda/4.8/open-source/tutorial-cordapp.md) to see Corda in action.
```